src/turret.cpp

```diff
@@ -513,6 +513,10 @@ int vehicle::automatic_fire_turret( vehicle_part &pt )
 
     int shots = 0;
 
+    if ( gun.query() != turret_data::status::ready ) {
```

Comment:
Astyle error - should be no whitespace after `if`:

![image](https://user-images.githubusercontent.com/16213433/39870528-696faa9c-546a-11e8-9a59-a41cf9d20f7a.png)